homeassistant/components/plex/strings.json

```diff
@@ -1,20 +1,35 @@
 {
   "config": {
     "step": {
+      "user": {
+        "title": "Plex Media Server",
+        "description": "Continue to <a href=\"https://plex.tv\" target=\"_blank\">plex.tv</a> to link a Plex server.",
```

Comment:
This description seems weird. Can we make it so that `manual_setup` is not a checkbox but that we offer 2 options:

 - Configure by logging in to Plex.tv
 - Manually enter server details

Reply:
So we use `vol.In`. We can look at ha-form to see if we can render radio buttons if < 5 options or so.

Reply:
The manual setup option will only be visible to those with advanced mode enabled. Most users will only see the description and a button to continue.

I'm trying to suggest in the language that proceeding is the recommended option and that a manual config is not.

Reply:
Ah yes, the description is shared between normal and advanced mode :( We could change that by using different steps ?

Reply:
That should be doable.